### PR TITLE
add leaderboadr component

### PR DESCRIPTION
**Pull request by SepsiLaszlo**

### CODE REVIEW

**Review by triszt4n — COMMENTED** [review]
Am I seeing it well, is your local config committed too? (application-x.properties)

It's not a problem, so if there's no time, theres no need to delete it from the PR

**Comment by SepsiLaszlo** [comment]
@triszt4n thanks for noticing, I removed the configfiles!

File: frontend/src/pages/leader-board/components/LeaderBoardListIem.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      <Flex align="center" justifyContent="space-between">
+        <Text fontWeight="bold" fontSize="xl">
+          {leaderBoardItem.name}
+        </Text>
+        <Text>{leaderBoardItem.score}</Text>
+        <ArrowRightIcon></ArrowRightIcon>
+      </Flex>
+    </Box>
+  )
+}
```
Comment (Tschonti) [inline frontend/src/pages/leader-board/components/LeaderBoardListIem.tsx:19]: this file isn't used anywhere

File: frontend/src/pages/leader-board/leaderBoard.page.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import { Box, Divider, Flex, GridItem, Hide, HStack, Link, SimpleGrid, useColorModeValue, VStack, Text } from '@chakra-ui/react'
+import { LeaderBoardListItem } from './components/LeaderBoardListIem'
+import { ArrowDownIcon, ArrowRightIcon, ChevronDownIcon, ChevronUpIcon } from '@chakra-ui/icons'
+import { useLeaderBoardQuery } from '../../api/hooks/useLeaderBoardQuery'
+import { LeaderBoardView } from '../../util/views/leaderBoardView'
+import { useConfigContext } from '../../api/contexts/config/ConfigContext'
+import { Helmet } from 'react-helmet-async'
+
+const LeaderboardPage = () => {
+  const { data } = useLeaderBoardQuery(() => console.log('Event query failed!'))
```
Comment (Tschonti) [inline frontend/src/pages/leader-board/leaderBoard.page.tsx:11]: Leaderboard query

File: frontend/src/pages/leader-board/leaderBoard.page.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  const userHoverBg = useColorModeValue('purple.300', 'purple.300')
+
+  const userBgDark = useColorModeValue('purple.600', 'purple.600')
+  const userHoverBgDark = useColorModeValue('purple.400', 'purple.400')
+
+  const groupBg = useColorModeValue('pink.400', 'pink.400')
+  const groupHoverBg = useColorModeValue('pink.300', 'pink.300')
+
+  const groupBgDark = useColorModeValue('pink.600', 'pink.600')
+  const groupHoverBgDark = useColorModeValue('pink.400', 'pink.400')
```
Comment (Tschonti) [inline frontend/src/pages/leader-board/leaderBoard.page.tsx:23]: I think we should use brand colors instead

File: frontend/src/pages/leader-board/leaderBoard.page.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        <GridItem colSpan={{ sm: 1, md: 2 }}>
+          <Divider bg="gray.400" height={2}></Divider>
+        </GridItem>
+
+        {data?.userBoard && data?.userBoard?.length > 0 && (
+          <VStack align="stretch" spacing={5}>
+            <Box id="user_scores" bg={userBgDark} px={6} py={4} borderRadius="md" _hover={{ bgColor: userHoverBgDark }}>
+              <Flex align="center" justifyContent="space-between">
+                <Text fontWeight="bold" fontSize="xl">
+                  Egyéni toplist
```
Comment (Tschonti) [inline frontend/src/pages/leader-board/leaderBoard.page.tsx:82]: typo

File: frontend/src/util/paths.ts [hunk 5]
```diff
@@ -11,7 +11,8 @@ export enum Paths {
   COMMUNITIES = 'communities',
   ERROR = 'error',
   EXTRA_PAGE = 'page',
-  FORM = 'form'
+  FORM = 'form',
+  LEADER_BOARD = 'leader-board'
```
Comment (Tschonti) [inline frontend/src/util/paths.ts:15]: leaderboard is one word, no hyphen needed

**Review by Tschonti — COMMENTED** [review]
🥇

**Pull request merged**